packages/manager/apps/ips/src/pages/listing/index.tsx

```diff
@@ -76,27 +146,36 @@ export default function Listing() {
   };
 
   return (
-    <BaseLayout header={header}>
-      <OdsButton
-        className="mb-5"
-        variant={ODS_BUTTON_VARIANT.outline}
-        icon={ODS_ICON_NAME.plus}
-        onClick={() => navigate(urls.order)}
-        label={t('orderIps')}
-      />
-      <React.Suspense fallback={<Loading />}>
-        {columns && flattenData && (
+    <RedirectionGuard
+      isLoading={isLoading || !ipList}
+      condition={!isLoading && !ipList.length}
+      route={urls.onboarding}
+      isError={isError}
+      errorComponent={
+        <OdsMessage className="mt-4" color={ODS_MESSAGE_COLOR.danger}>
+          {tError('manager_error_page_default')}
+        </OdsMessage>
```

Comment:
You can put the ErrorBanner directly and remove the `if(isLoading)` and `if(isError)` above